pyproject.toml

```
@@ -4,6 +4,32 @@ version = "2024.12.2"
 description = ""
 authors = ["authentik Team <hello@goauthentik.io>"]
 
+[tool.bandit]
+exclude_dirs = ["**/node_modules/**"]
+
+[tool.codespell]
+skip = [
+    "**/node_modules",
+    "**/package-lock.json",
+    "authentik",
+    "schema.yml",
+    "schema.yaml",
+    "schema.json",
+    "go.sum",
+    "cmd",
+    "internal",
+    "web/src",
+    "locale",
+    "**/web/xliff/**",
+    "./website/blog/**",
+    "./website/docs/**",
+    "./website/integrations/**",
+    "./website/src/**",
```

```suggestion
    "locale",
    "**/web/xliff/**",
    "./website/src/**",
```

Many of those we definitely don't want to skip. It's nice to catch spelling mistakes in the code. You can also remove `authentik` from that list.

On top of that, I would say that this might cause more problems down the line with generated files from the tools we use (like `__pycache__`, `gen-go-api`, `gen-ts-api`, `unittest.xml` to name a few`). I would rather go with the previous approach of including what needs to be included and leaving the rest alone.

---

But then again, we were missing a few things like `scripts`. If codespell could ignore the same things as .gitignore, plus a few generated committed things that'd be great, if we can reuse the .gitignore file, let's not duplicate that list